app/build.gradle

```diff
@@ -29,6 +29,7 @@ dependencies {
     })
     compile 'com.android.support:appcompat-v7:25.1.0'
     testCompile 'junit:junit:4.12'
-    compile 'com.android.volley:volley:1.0.0'
```

Comment:
@mozgaanna just fyi this was also done in @radaladar 's PR, you dont have to do things twice